test/test_helper.rb

```diff
@@ -74,6 +74,7 @@ def create_items_associated_with_orders
     order_2 = Order.create(current_status: 'completed')
     order_3 = Order.create(current_status: 'paid')
     order_4 = Order.create(current_status: 'canceled')
+    order_5 = Order.create(current_status: 'ordered')
```

Comment:
Useless assignment to variable - `order_5`. Did you mean `order`?<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.